src/prng/isaac.rs

```diff
@@ -10,53 +10,97 @@
 
 //! The ISAAC random number generator.
 
-#![allow(non_camel_case_types)]
-
 use core::slice;
 use core::iter::repeat;
 use core::num::Wrapping as w;
 use core::fmt;
 
 use {Rng, FromRng, SeedableRng};
 
-/// Select 32- or 64-bit variant dependent on pointer size.
-#[cfg(target_pointer_width = "32")]
-pub use prng::IsaacRng as IsaacWordRng;
-#[cfg(target_pointer_width = "64")]
-pub use prng::Isaac64Rng as IsaacWordRng;
-
-#[allow(bad_style)]
-type w64 = w<u64>;
-#[allow(bad_style)]
+#[allow(non_camel_case_types)]
 type w32 = w<u32>;
 
 const RAND_SIZE_LEN: usize = 8;
-const RAND_SIZE: u32 = 1 << RAND_SIZE_LEN;
-const RAND_SIZE_USIZE: usize = 1 << RAND_SIZE_LEN;
+const RAND_SIZE: usize = 1 << RAND_SIZE_LEN;
 
-/// A random number generator that uses the ISAAC algorithm[1].
+/// A random number generator that uses the ISAAC algorithm.
+///
+/// ISAAC stands for "Indirection, Shift, Accumulate, Add, and Count" which are
+/// the principal bitwise operations employed. It is the most advanced of a
+/// series of array based random number generator designed by Robert Jenkins
+/// in 1996[1][2].
+///
+/// Although ISAAC is designed to be cryptographically secure, its design is not
+/// founded in cryptographic theory. Therefore it should _not_ be used for
+/// cryptographic purposes. It is however one of the strongest non-cryptograpic
+/// RNGs, and that while still being reasonably fast.
+///
+/// Where fast random numbers are needed which should still be secure, but where
+/// speed is more important than absolute (cryptographic) security (e.g. to
+/// initialise hashes in the std library), a generator like ISAAC should be
```

Comment:
*should* is very assertive language. Unless you're sure about this it might be better to write something like "may be a good choice".

Reply:
I have copied this line from your RFC ;-)